src/iosrv/init.c

```diff
@@ -611,10 +612,57 @@ server_init(int argc, char *argv[])
 	return rc;
 }
 
+static int
+cleanup_cb(const char *path, const struct stat *sb, int flag,
+	   struct FTW *ftwbuf)
+{
+	int rc;
+
+	if (ftwbuf->level == 0)
+		return 0;
+
+	if (flag == FTW_DP || flag == FTW_D)
+		rc = rmdir(path);
+	else
+		rc = unlink(path);
+	if (rc)
+		D_ERROR("failed to remove %s\n", path);
+	return rc;
```

Comment:
Do we need to convert the standard library return value + errno to -DER_* values? There are more such places below in this PR.